### PR TITLE
fix(wallet): direct Edit Details btn to editable pubkey page

### PR DESCRIPTION
**Pull request by honoka428**
ISSUES: #109

## Description

Bug fix for an open issue (#109) where the Edit Details button would clear the wallet when clicked. It now directs back to the editable xpubs page.

caravan-pr

## Does this PR introduce a breaking change?

* [ ] Yes
* [X] No

## Does this PR fix an open issue?

* [X] Yes
* [ ] No
#109

### CODE REVIEW

File: src/components/Wallet/WalletGenerator.jsx [hunk 1]
```diff
@@ -143,18 +143,20 @@ class WalletGenerator extends React.Component {
...
       resetWallet,
+      initialLoadComplete,
       setGenerating,
     } = this.props;
 
     if (!configuring) {
       setGenerating(false);
-      resetWallet();
+      if (type === "edit") initialLoadComplete();
+      if (type === "clear") resetWallet();
```
Comment (waldenraines) [inline src/components/Wallet/WalletGenerator.jsx:159]: Perhaps use a boolean instead?  

```js
toggleImporters = (event, clear=true)
```

**Review by waldenraines — CHANGES_REQUESTED** [review]
@honoka428 thanks for the Pull Request!

I just have on minor comment, otherwise this seems fine.

File: src/components/Wallet/WalletGenerator.jsx (hunk 1 above)
Comment (honoka428) [reply src/components/Wallet/WalletGenerator.jsx]: Thanks for the review, @waldenraines! Switched it to a boolean as suggested.

**Review by bucko13 — APPROVED** [review]
Thanks for the contribution! LGTM

**Review by waldenraines — APPROVED** [review]
LGTM, thanks for the contribution!

**Pull request merged**